examples/C++/src/demo3.cxx

```diff
@@ -54,7 +55,8 @@ class MyStuff: public ParsingObject
 void
 MyStuff::set_defaults()
 {
-  back_projector_sptr.reset(new BackProjectorByBinUsingInterpolation);
+  shared_ptr<ProjMatrixByBin> PM(new  ProjMatrixByBinUsingRayTracing());
+  back_projector_sptr.reset(new BackProjectorByBinUsingProjMatrixByBin(PM)); 
```

Comment:
Can this be done in one line? Maybe it is best to leave as two

Reply:
can be done in 1 line, but could become a bit cryptic. Could use the `auto` trick from above